performance-tests/remotewb-performance-tests/addUsers.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+$JBOSS_HOME/bin/add-user.sh -a perfUser 'perfUser1234;'
+$JBOSS_HOME/bin/add-user.sh -a engUser 'engUser1234;'
+echo "perfUser=admin" >> $JBOSS_HOME/standalone/configuration/application-roles.properties
+echo "engUser=admin,engineering" >> $JBOSS_HOME/standalone/configuration/application-roles.properties
```

Comment:
Something like [this](https://github.com/droolsjbpm/kie-tests/blob/master/remote-kie-tests/kie-wb-tests/kie-wb-tests-eap/pom.xml#L365-L384)? 

Basically, you generate the properties once, store them in a directory, and then copy them over the default ones using the resources plugin.